src/lsdb/io/to_hipscat.py

```diff
@@ -119,16 +121,28 @@ def write_partitions(
     pixel_to_result_index = {}
 
     partitions = catalog._ddf.to_delayed()
+
     for index, (pixel, partition_index) in enumerate(catalog._ddf_pixel_map.items()):
         results.append(
-            perform_write(partitions[partition_index], pixel, base_catalog_dir_fp, storage_options, **kwargs)
+            perform_write(
+                partitions[partition_index],
+                pixel,
+                base_catalog_dir_fp,
+                storage_options,
+                **kwargs,
+            )
         )
         pixel_to_result_index[pixel] = index
 
     partition_sizes = dask.compute(*results)
 
+    if all(size == 0 for size in partition_sizes):
+        raise AssertionError("The output catalog is empty")
```

Comment:
Can this be changed from AssertionError? Maybe something a runtime error instead?

Reply:
Sure, just pushed the changes!